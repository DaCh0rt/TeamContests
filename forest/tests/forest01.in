7 958
5 0
5 5
0 0
2 1
1 1
3 2
0 5
