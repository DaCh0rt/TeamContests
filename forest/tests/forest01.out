6039.29 2902425.64
